### DISCUSSION TITLE
What does this update?

### DISCUSSION THREAD

**Discussion by huskylogic** [NexovaDev/UpdateHub#249] [Q&A] [answered]
I see tools like UCheck, iobit update, and others. They have a list of software they can check updates for. If the software is not on this list, it won't look. 

Is this program the same way? Does it have a compatible software list, or is it like the old file hippo program where it scans for what software is on your computer and then looks for any updates? In other words there's no limit as to what software it will or won't look for updates for?

**Comment by onyx-nxt** [comment, marked as answer]
There's a giant repository with all the latest version info that UpdateHub checks and compares that version with the installed app on your computer. If the app is in that repository, then UpdateHub can scan it for updates
